docs/decorator-reference.md

```diff
@@ -19,15 +19,15 @@
     * [`@JoinColumn`](#joincolumn)
     * [`@JoinTable`](#jointable)
     * [`@RelationId`](#relationid)
-* [Subscriber and listener decorators](#subscriber-and-listener-decorators)
-    * [`@AfterLoad`](#afterload)
-    * [`@BeforeInsert`](#beforeinsert)
-    * [`@AfterInsert`](#afterinsert)
-    * [`@BeforeUpdate`](#beforeupdate)
-    * [`@AfterUpdate`](#afterupdate)
-    * [`@BeforeRemove`](#beforeremove)
-    * [`@AfterRemove`](#afterremove)
-    * [`@EventSubscriber`](#eventsubscriber)
+* [Subscriber and listener decorators](./listeners-and-subscribers.md)
```

Comment:
I know its a duplication, but I really wanted to keep all decorators on a single page

Reply:
Okay, I will revert this change